web3/manager.py

```diff
@@ -113,12 +113,12 @@ def request_blocking(self, method, params):
 
```

Comment:
This change is only a fix for web3 request_async method, is not being used anymore in this solution, we can remove it but actually it fixes a typo in web3 since the parameters in the current version are wrong, anyway if you think we should remove it i can rollback these changes